scipy/stats/_survival.py

```diff
@@ -442,29 +443,39 @@ def logrank(
 
     Notes
     -----
-    The logrank test [1]_ compares the expected events that occured at
```

Comment:
I don't think "expected events that occured" is appropriate here. This part is just the *observed* number of events; the next part is the *expected* number of events.